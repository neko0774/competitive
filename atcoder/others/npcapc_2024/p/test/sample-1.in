4
4
131
333854876
620963539
