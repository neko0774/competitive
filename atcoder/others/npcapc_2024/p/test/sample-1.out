4 2
1 3 3 2
6 3
6 2 6 4 5 1 
17 10
8 7 1 14 8 5 12 5 2 17 7 16 15 17 14 15 6 
24 14
16 8 14 18 15 16 21 4 10 14 3 4 22 8 16 11 22 17 11 5 12 2 13 18 
